packages/scene-composer/src/components/panels/scene-components/data-overlay/DataOverlayPanelConfigEditor.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useCallback } from 'react';
+import { Checkbox, FormField } from '@awsui/components-react';
+import { useIntl } from 'react-intl';
+
+import { IDataOverlayComponentInternal } from '../../../../store';
+import { Component } from '../../../../models/SceneModels';
+
+interface IDataOverlayPanelConfigEditorProps {
+  config: Component.OverlayPanelConfig | undefined;
+  onUpdateCallback: (componentPartial: Partial<IDataOverlayComponentInternal>, replace?: boolean | undefined) => void;
+}
+
+export const DataOverlayPanelConfigEditor: React.FC<IDataOverlayPanelConfigEditorProps> = ({
+  config,
+  onUpdateCallback,
+}) => {
+  const { formatMessage } = useIntl();
+
+  const onPinnedChange = useCallback(
+    ({ detail: { checked } }) => {
+      onUpdateCallback({ config: { isPinned: checked } }, false);
+    },
+    [onUpdateCallback],
+  );
+
+  return (
+    <FormField label={formatMessage({ defaultMessage: 'Panel Config', description: 'FormField label' })}>
```

Comment:
FormField labels are inconsistent throughout the package. Are these verified with Tech writer later?

Reply:
I will review with UX later, I don't have any mock yet right now, so I'm basically just making up with an UI that let me input all supported configurations